portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -79,28 +76,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   }
 
   @Override
-  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException{
-
-      information = checkValidity(request);
-
-      // Send invalid input to client for correction
-      if (((Boolean) information.get("nameInfo").get("error") == true) || 
-          ((Boolean) information.get("messageInfo").get("error") == true)){
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    information = checkValidity(request);
 
-      } else {
+    // Send invalid input to client for correction
+    if (((Boolean) information.get("nameInfo").get("error") == true)
+        || ((Boolean) information.get("messageInfo").get("error") == true)) {
```

Comment:
Duplicate code from #8 .

Reply:
done

Reply:
Was this done? I still see == true. Will #8 fix this?